Automattic/wp-calypso
# Stats  polish: Posting Activity UI in the Insights page

#### Proposed Changes

* Update the Posting Activity design
* Fix the position of the days (from vertical to horizontal)

#### Testing Instructions



* Go to the Calypso live link
* Navigate to the Insights tab
* Ensure the current site has data available for the Posting Activity section
* Ensure the Posting Activity section is following the design specs

#### Screenshots

Before:
![image](https://user-images.githubusercontent.com/6406071/210022760-338c0794-2961-403d-96f4-fba582e23556.png)
![image](https://user-images.githubusercontent.com/6406071/210022772-56f5edbe-f19a-4d8e-ae93-de25f66ed592.png)

After:
![image](https://user-images.githubusercontent.com/6406071/210022797-ff39cd14-1eaa-42b2-9ea5-d5e65634abd8.png)
![image](https://user-images.githubusercontent.com/6406071/210022807-373b32dc-7017-4762-8d3e-7d52b6f72764.png)

#### Pre-merge Checklist





Related to #70609
